tcrypto/sys/src/lib.rs

```diff
@@ -21,41 +21,3 @@ pub use mc_sgx_tcrypto_sys_types::{
 };
 
 include!(concat!(env!("OUT_DIR"), "/bindings.rs"));
```

Comment:
See https://github.com/intel/linux-sgx/issues/958 this is a function meant to be used from the trusted side.
We don't currently have a way to unit test the trusted side of things, we were getting by calling into this from outside of an enclave, but don't think it's worth the extra work to pull in more C/C++ dependencies to be able to test the C bindings
This was one of the few places that we were testing the bindings to a trusted interface.